compiler/bindings/c/iree/compiler/embedding_api.h

```diff
@@ -224,6 +224,13 @@ IREE_EMBED_EXPORTED bool
 ireeCompilerInvocationParseSource(iree_compiler_invocation_t *inv,
                                   iree_compiler_source_t *source);
 
+// Sets a mnemonic phase name to run compilation from. Default is "input".
+// The meaning of this is pipeline specific. See IREEVMPipelinePhase
+// for the standard pipeline.
+IREE_EMBED_EXPORTED void
```

Comment:
Can you increase the IREE_COMPILER_API_MINOR version to 3 and add a note here: // Available since 1.3

Reply:
(note that the version is in Embed.cpp)